Jenkinsfile

```diff
@@ -139,6 +141,10 @@ pipeline {
         }
         stage('make test-with-kind') {
           steps {
+            script {
```

Comment:
```suggestion
            environment {
```

block should work or just override them on ”make” directly (see BUILDTAGS example)

Reply:
seems not working..
let me do same as BUILDTAGS